Doppler.HtmlEditorApi/Domain/DopplerHtmlDocument.cs

```diff
@@ -30,11 +30,11 @@ public class DopplerHtmlDocument
     private const string FIELD_ID_TAG_START_DELIMITER = "|*|";
     private const string FIELD_ID_TAG_END_DELIMITER = "*|*";
     // &, # and ; are here to accept HTML Entities
-    private static readonly Regex FIELD_NAME_TAG_REGEX = new Regex($@"{Regex.Escape(FIELD_NAME_TAG_START_DELIMITER)}([a-zA-Z0-9 \-_ñÑáéíóúÁÉÍÓÚ%&;#]+){Regex.Escape(FIELD_NAME_TAG_END_DELIMITER)}");
-    private static readonly Regex FIELD_ID_TAG_REGEX = new Regex($@"{Regex.Escape(FIELD_ID_TAG_START_DELIMITER)}(\d+){Regex.Escape(FIELD_ID_TAG_END_DELIMITER)}");
-    private static readonly Regex CLEANUP_URL_REGEX = new Regex(@"^(?:\s?(?:%20)?)*|(?:\s?(?:%20)?)*$|\s");
-    private static readonly Regex TRACKABLE_URL_ACCEPTANCE_REGEX = new Regex(@"^(?:\s?(?:%20)?)*(?:(?:https?|ftp):\/\/|www\.)", RegexOptions.IgnoreCase);
-    private static readonly Regex TRACKABLE_URL_PARTS_REGEX = new Regex(@"^(?:(?<scheme>(?:https?|ftp):\/\/)(?<domain>[^\/]+)|(?<domainWithoutScheme>www\.[^\/]+))(?<rest>\/.*)?$", RegexOptions.IgnoreCase);
+    private static readonly Regex _fieldNameTagRegex = new Regex($@"{Regex.Escape(FIELD_NAME_TAG_START_DELIMITER)}([a-zA-Z0-9 \-_ñÑáéíóúÁÉÍÓÚ%&;#]+){Regex.Escape(FIELD_NAME_TAG_END_DELIMITER)}");
+    private static readonly Regex _fieldIdTagRegex = new Regex($@"{Regex.Escape(FIELD_ID_TAG_START_DELIMITER)}(\d+){Regex.Escape(FIELD_ID_TAG_END_DELIMITER)}");
+    private static readonly Regex _cleanupUrlRegex = new Regex(@"^(?:\s?(?:%20)?)*|(?:\s?(?:%20)?)*$|\s");
+    private static readonly Regex _trackableUrlAcceptanceRegex = new Regex(@"^(?:\s?(?:%20)?)*(?:(?:https?|ftp):\/\/|www\.)", RegexOptions.IgnoreCase);
+    private static readonly Regex _trackableUrlPartsRegex = new Regex(@"^(?:(?<scheme>(?:https?|ftp):\/\/)(?<domain>[^\/]+)|(?<domainWithoutScheme>www\.[^\/]+))(?<rest>\/.*)?$", RegexOptions.IgnoreCase);
```

Comment:
I do not feel comfortable with this.

![image](https://user-images.githubusercontent.com/1157864/162955277-3c70479d-0aff-4a98-8666-4b7e93d5ac9d.png)

The _real_ constants are written in all upper-case, while the values that are constant (because they never change) are written in underscore-prefixed camel-case. 

Maybe we should use pascal-case for all of them?

Something like this:

* private fields → _camelCase
* private read-only fields → _camelCase
* public static read-only fields → PascalCase
* public static read-only fields → PascalCase
* private consts → PascalCase
* public consts → PascalCase

What do you think?

Reply:
In this case:

```csharp
    // . . .
    private const string FieldNameTagStartDelimiter = "[[[";
    private const string FieldNameTagEndDelimiter = "]]]";
    private const string FieldIdTagStartDelimiter = "|*|";
    private const string FieldIdTagEndDelimiter = "*|*";
    private static readonly Regex FieldNameTagRegex = new Regex($@"{Regex.Escape(FIELD_NAME_TAG_START_DELIMITER)}([a-zA-Z0-9 \-_ñÑáéíóúÁÉÍÓÚ%&;#]+){Regex.Escape(FIELD_NAME_TAG_END_DELIMITER)}");
    private static readonly Regex FieldIdTagRegex = new Regex($@"{Regex.Escape(FIELD_ID_TAG_START_DELIMITER)}(\d+){Regex.Escape(FIELD_ID_TAG_END_DELIMITER)}");
    private static readonly Regex CleanupUrlRegex = new Regex(@"^(?:\s?(?:%20)?)*|(?:\s?(?:%20)?)*$|\s");
    private static readonly Regex TrackableUrlAcceptanceRegex = new Regex(@"^(?:\s?(?:%20)?)*(?:(?:https?|ftp):\/\/|www\.)", RegexOptions.IgnoreCase);
    private static readonly Regex TrackableUrlPartsRegex = new Regex(@"^(?:(?<scheme>(?:https?|ftp):\/\/)(?<domain>[^\/]+)|(?<domainWithoutScheme>www\.[^\/]+))(?<rest>\/.*)?$", RegexOptions.IgnoreCase);

    private readonly HtmlNode _headNode;
    private readonly HtmlNode _contentNode;
    // . . .
```

Reply:
I like it this way!!

Reply:
Updated! PascalCase for all!